library/spdm_crypt_lib/libspdm_crypt_cert.c

```diff
@@ -1131,8 +1161,10 @@ bool libspdm_x509_common_certificate_check(const uint8_t *cert, size_t cert_size
     if (LIBSPDM_CRYPTO_X509_KU_DIGITAL_SIGNATURE & value) {
         status = true;
     } else {
-        status = false;
-        goto cleanup;
+        if (cert_model != SPDM_CERTIFICATE_INFO_CERT_MODEL_GENERIC_CERT) {
```

Comment:
Regardless of model, if `Key Usage` is present then digital signature must be set. In this logic if
- Certificate model is generic.
- Key usage is present.
- Digital signature bit is not set.

then the function will return `true` instead of `false`.

Reply:
Thanks. I have updated the logic:
```
    if (!status) {
        goto cleanup;
    } else {
        if (value == 0) {
            if (cert_model != SPDM_CERTIFICATE_INFO_CERT_MODEL_GENERIC_CERT) {
                status = false;
                goto cleanup;
            }
        } else {
            if ((LIBSPDM_CRYPTO_X509_KU_DIGITAL_SIGNATURE & value) == 0) {
                status = false;
                goto cleanup;
            }
        }
    }
```